scripts/optimize_zshrc.sh

```
@@ -0,0 +1,72 @@
+#!/bin/bash
+# 🔧 Оптимизация .zshrc для быстрой загрузки
+# Usage: ./scripts/optimize_zshrc.sh
+
+set -euo pipefail
+
+ZSHRC_FILE="$HOME/.zshrc"
+PROJECT_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+ALIASES_SCRIPT="$PROJECT_ROOT/setup_cli_aliases.sh"
```

Shellcheck (0.11.0)

[warning] ALIASES_SCRIPT appears unused. Verify use (or export if used externally).

(SC2034)